embassy-stm32/build.rs

```diff
@@ -189,6 +189,7 @@ fn main() {
         Some("tim5") => "TIM5",
         Some("tim8") => "TIM8",
         Some("tim9") => "TIM9",
+        Some("tim11") => "TIM11",
```

Comment:
ah, I made a mistake. I actually remove TIM11 from time-driver last time, since this not capable be a time-driver. TIM11 only have 1 CC channel, but time-driver require at least 2. Would you remove it, and also remove it from Cargo.toml?

Reply:
Maybe add comment about TIM11 being unusable due to reasons mentioned above.